src/services/DeliverablesService.ts

```diff
@@ -17,38 +19,46 @@ const SEARCH_FIELDS_DELIVERABLES_BASE = ['documentCount', 'id', 'name', 'project
 
 const SEARCH_FIELDS_DELIVERABLES_ADMIN = [...SEARCH_FIELDS_DELIVERABLES_BASE, 'category', 'description'];
 
+const mockDeliverable: Deliverable = {
+  id: 1,
+  documents: [
+    {
+      name: 'Upload 1',
+      description: 'Some description for the upload',
+      dateUploaded: '2024-02-15',
+    },
+  ],
+  category: 'Legal',
+  status: 'Not Submitted',
+  name: 'Company Formation Document',
+  projectName: 'Treemendo.us',
+  projectId: 1,
+  // TODO need to figure out if we are going to allow raw HTML or just have regular text
+  deliverableContent:
+    'The Company Formation Document is to confirm the entity is properly formed and registered in the country.\n' +
+    '\n' +
+    'Depending on your jurisdiction, this document may be called a Certificate of Incorporation or a Business Registration Certificate, for example.\n' +
+    '\n' +
+    'Submit:\n' +
+    'A document issued by the relevant authority in your jurisdiction that contains the following information:\n' +
+    'Company Name\n' +
+    'Legal Form / Structure – (e.g. corporation, limited liability company, etc.)\n' +
+    'Date of Formation/Incorporation\n' +
+    'Company Number (if applicable)',
+};
+
 const getDeliverable = async (deliverableId: number): Promise<Response & DeliverableData> => {
   // TODO replace with axios
-  return {
+  return Promise.resolve({
```

Comment:
Not that it matters since this is mock data, but if you put `async` in the function declaration it will automatically wrap your return in a promise, so you can just `return { ... }` 🚀

Reply:
Yes, that was my understanding. I was trying some things to mock a delay (to test the spinner) - will remove this.